app/controllers/v1/nominations_controller.rb

```diff
@@ -0,0 +1,36 @@
+class V1::NominationsController < V1::BaseController
+  FORM_ID = '1sQtLTyZWA6KDsi7-ToB2VyOGaz44MZEHQ_fUvlX8VQ0'
+
+  MAPPINGS = {
+    legislator_id:     'entry.353543474',
+    selection_comment: 'entry.1466634721',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.